go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/networkservicemesh/integration-k8s-gke
+
+go 1.15
```

Comment:
Let's use go 1.16

Reply:
Previously I've updated `integration-tests` and `integration-k8s-kind` so now it is critical.

Reply:
Done